api/cloud/oc_cloud_manager.c

```diff
@@ -587,10 +613,15 @@ cloud_manager_login_handler(oc_client_response_t *data)
   oc_cloud_context_t *ctx = (oc_cloud_context_t *)data->user_data;
   oc_remove_delayed_callback(ctx, cloud_manager_login_async);
   bool handleUnauthorizedByRefresh = cloud_context_has_refresh_token(ctx);
```

Comment:
Since `_login_handler` no longer calls `cloud_context_clear`, we no longer need to check if the refresh token is in the context here, we can move this context check to where `handleUnauthorizedByRefresh` is used (line 650)

Reply:
moved